confidant/cli/environments.py

```
@@ -0,0 +1,160 @@
+# confidant/cli/environments.py
+
+import os
+import shutil
+import typer
+import yaml
```

Ruff (0.8.2)

`yaml` imported but unused

Remove unused import: `yaml`

(F401)